### DISCUSSION TITLE
[Design/General] Two new ghost roles, the simulacrum and "off station"

### DISCUSSION THREAD

**Discussion by hal9000PR** [ParadiseSS13/Paradise#17796] [Documents]
## Goals

- Reduce player aversion to death.
- Open up opportunities for people to learn antagonist abilities/items/strategies in a controlled environment.
- Allow people to generally learn departments without the need to mess up on the job. (reducing malpractice bay and similar such roles.
- Give dead players something more to do when a round is slow.

## The ideas
The simulacrum is an area in were you can load in items, structures, and other such gear. 
Players would be able to practice using antagonist items/abilities without the major anxiety of doing it for the first time. 
 - Hopefully this will reduce the number of wizards, nuke ops, cultists, etc. that have no clue how to play.

 - Players could also use this area to test combat skills against each other.

Then, "off station" (needs a better name) would be an area of RP related areas... off station. where you can roleplay as your character not in a work environment. This gives dead players, who still want to roleplay a place to go. Hopefully this means that death is very final for your character on station, but you can still continue to play.

- Places like bars, restaurants, cinemas, shops. it should feel like a town centre. 

## Initial Concerns
- if your friend dies IC, instead of RPing it out, you could just cryo and go to the "off station" there, removing a lot of OOC stakes. However that is the consequence of reducing players OOC aversion to death.

- "the test server exists for this purpose" many players don't know this exists, and its not exactly easy to go around and mess about on there without getting admin perms. having it in game makes it 100% accessible to anyone who ever dies or clicks observe.

- Players might potentially find a way to find a radio headset and contact the actual station from either of these roles. which would break immersion. 

- A similar system was already tried with VR. I hope that making this completely separate from the station will fix some of the issues that were presented by VR.

**Comment by Sirryan2002** [comment]
I have a few concerns with the off station area. What's to stop players from going here every round instead of chilling out on station? I have no doubt that there is a large group of players on our server who will avoid joining into the normal round because the off-station area is better: 1) because it will mostly be non-violent and 2) it is designed to be "comfy" as possible (as in geared for RP and an enjoyable/fun area to chill in) without the interruptions of the normal station. This is largely a subjective problem as a lot of people might not see this as an actual issue (rather a benefit as well), however, I'm worried this will not only take away "bridge hobos" but also people who will chill out in the bar, dorms, or their own little custom made place on station. TL;DR this is moving roleplayers off the station.

As for the Simulacrum, I think this is a really awesome idea and it will be a super engaging ghost role. I think it will fulfill its purpose as a testing ground really well and players won't be glued to it (because it eventually gets repetitive after a while). I agree with the caveats you identified with it, although I'm sure item blacklists and smart implementation will largely negate these issues.

**Reply by hal9000PR** [reply]
You could limit the ghost role to people who've actually died. Not cryoed, observed or suicided. Or maybe let observers/cryoers join after a delay? Maybe 30 minutes? That way you don't just join to go to the RP den.
